### DISCUSSION TITLE
Playtime 2: Be able to save window size as default and select scaling

### DISCUSSION THREAD

**Discussion by joungle123** [helgoboss/helgobox#1279] [Ideas]
Be able to save window size as default meaning every time we open Playtime, the window size remains the one we resized last time or at least somehow to select window size and scaling as an option in preferences.

**Comment by BradleyBemis** [comment]
I agree, this would be great! 🙂

**Comment by helgoboss** [comment]
I suppose you are talking about opening a new Playtime instance? Because when opening existing ones, that should work already.

**Reply by joungle123** [reply]
@helgoboss Yes, when starting a new project in Reaper and opening a new instance of Playtime, so that we would not need to resize or scale every time we open a new instance and project.
